src/ops/math.jl

```diff
@@ -99,15 +99,28 @@ end
 ^(n::AbstractTensor, x::Int) = invoke(^, (AbstractTensor, Any), n, x)
 .^(n::AbstractTensor, x::Number) = n^x
 
+for (jl_func_name, tf_func_name) in [
+    (:neg, "Neg"),
+    (:square, "Square"),
+    (:shape, "Shape")]
+    @eval function $jl_func_name(n::AbstractTensor; name=$tf_func_name)
+        local desc
+        with_op_name(name) do
+            n = Tensor(n)
+            name = get_name(name)
+            desc = NodeDescription($tf_func_name, name)
```

Comment:
Eliminate the `name` argument since the name has already been established by the `with_op_name` call